src/main/java/com/brightsparklabs/asanti/AsantiCli.java

```diff
@@ -57,10 +65,24 @@ public static void main(String[] args)
     {
         try
         {
-            switch (args.length)
+            Options options = getOptions();
+            CommandLineParser parser = new DefaultParser();
+            final CommandLine cmdLine = parser.parse(options, args);
+            validateCommandLine(cmdLine);
+
+            if (cmdLine.hasOption("l"))
+            {
+                // Set log4j configuration
+                Properties properties = new Properties();
+                properties.load(new FileInputStream(cmdLine.getOptionValue("l")));
+                LogManager.resetConfiguration();
```

Comment:
I don't love this.

We are using slf4j as a logging facade, meaning we should be able to plug in different logging implementations at the build step without needing to change code.  By going directly to log4j I think we are undermining this - am I mistaken?